### PR TITLE
[FTRMT032022] Oscar Gonzalez

### PR DESCRIPTION
**Pull request by oscargonzalezdev**
- [x] solve lab javascript greatest movies

### CODE REVIEW

File: src/movies.js [hunk 1]
```diff
@@ -1,26 +1,80 @@
...
+  });
+  return directors;
+}
 
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
-function howManyMovies() {}
+function howManyMovies(movies) {
+  const times = movies.filter(item => item.director === 'Steven Spielberg');
+return times.length;
+}
```
Comment (TA-Remote) [inline src/movies.js:19]: Here you are asked to filter by two factors, by director and by genre , so missing one more check ;)

File: src/movies.js [hunk 2]
```diff
@@ -1,26 +1,80 @@
...
+    }, 0);
+    let average = Math.round((sum/movies.length) * 100) / 100;
+    return average;
+  }
 
 // Iteration 4: Drama movies - Get the average of Drama Movies
-function dramaMoviesScore() {}
+function dramaMoviesScore(movies) {
+  let dramaMovies = movies.filter(item => item.genre[0] === 'Drama');
+  return scoresAverage(dramaMovies);
```
Comment (TA-Remote) [inline src/movies.js:46]: Great - you are reusing the function from before

File: src/movies.js [hunk 3]
```diff
@@ -1,26 +1,80 @@
...
+
+  movies.forEach(item => {
+    if (typeof item.score != 'number'){
+      item.score = 0;
+    };
+  });
+
+  if (movies.score){
+    movies.score = 0;
+  }
```
Comment (TA-Remote) [inline src/movies.js:35]: in your function `movies` is an array, so lines 33-35 makes no sense, cause array of movies does not have property `score`.. Only objects have properties

You can just remove those lines - makes no difference to the code

**Review by TA-Remote — COMMENTED** [review]
Great work Oscar. You can review suggested solution for iteration6 in student portal :)

File: src/movies.js (hunk 1 above)
Comment (oscargonzalezdev) [reply src/movies.js:19]: I see. Thanks!

File: src/movies.js (hunk 3 above)
Comment (oscargonzalezdev) [reply src/movies.js:35]: I see the mistake ;)

**Pull request closed without merging**